src/cryptography/x509/name.py

```diff
@@ -114,11 +114,19 @@ def sub(m):
     return _RFC4514NameParser._PAIR_RE.sub(sub, val)
 
 
-class NameAttribute:
+NameAttributeValueType = typing.TypeVar(
+    "NameAttributeValueType",
+    typing.Union[str, bytes],
+    str,
+    bytes,
```

Comment:
Is it right that these all need to be listed? If you do only `str, bytes`, does it reject `str | bytes`  in the impl?

Reply:
Yes, only specifying `str, bytes` does not include `str | bytes` - and using `str | bytes` would break compatibility with Python 3.9 and break tests